lib/matplotlib/mpl-data/fonts/ttf/local.conf

```diff
@@ -1,32 +0,0 @@
-<?xml version="1.0"?>
-<!DOCTYPE fontconfig SYSTEM "fonts.dtd">
-<!-- /etc/fonts.conf file to configure system font access -->
-<fontconfig>
-        <!--  Enable sub-pixel rendering
-        <match target="font">
-                <test qual="all" name="rgba">
-                        <const>unknown</const>
-                </test>
-                <edit name="rgba" mode="assign"><const>rgb</const></edit>
-        </match>
-         -->
-
-        <alias>
-                <family>serif</family>
-                <prefer>
-                        <family>Bitstream Vera Serif</family>
-                </prefer>
-        </alias>
-        <alias>
-                <family>sans-serif</family>
-                <prefer>
-                        <family>Bitstream Vera Sans</family>
-                </prefer>
-        </alias>
-        <alias>
-                <family>monospace</family>
-                <prefer>
-                        <family>Bitstream Vera Sans Mono</family>
-                </prefer>
-        </alias>
-</fontconfig>
```

Comment:
@jenshnielsen: As @zblz said, since DejaVu is a superset of Vera, using DejaVu whenever Vera is requested should have *no* visible impact other than missing characters now being present, which I'd call a bugfix, not a behavior change.  I don't think there's any benefit to keeping the old font, particularly with font subsetting working so well in all backends.

That said, it might be worth adding a built-in mapping from Vera to DejaVu, so that if a user was explicitly setting Vera (rather than just leaving the defaults), and they don't have it installed system wide, they will get DejaVu and things continue to work.  This is the kind of thing where some extra release candidate testing may be necessary to catch all of the environmental gotchas about this change.